agimus_controller_ros/agimus_controller_ros/agimus_controller.py

```diff
@@ -47,7 +55,20 @@ def __init__(self, node_name: str = "agimus_controller_node") -> None:
         self.param_listener = agimus_controller_params.ParamListener(self)
         self.params = self.param_listener.get_params()
         self.params.ocp.armature = np.array(self.params.ocp.armature)
-        self.traj_buffer = TrajectoryBuffer([(1, self.params.ocp.horizon_size - 1)])
+        self.params.horizon_size_base_dt = 0
```

Comment:
Any reason not to make it an attribute of this class rather than `params`, whose class is generated by ROS.
```suggestion
        self._trajectory_buffer_size_for_horizon = 0
```
The name above is a suggestion to get rid of `horizon_size` which is highly subject to interpretations.

Reply:
I see that you renamed the parameter. That's good.

I would also like that the code does not modify the parameters but instead stores the value as a private attribute internally.